platforms/macosx/src/GDeviceOSX.cpp

```diff
@@ -600,113 +586,17 @@ void GDeviceOSX::DrawMusicSymbol( float x, float y, unsigned int inSymbolID )
 			x -= (w * float(0.5));
 	}
 
-	::CGContextSelectFont(mContext, mCurrMusicFont->GetName(), mCurrMusicFont->GetSize(), kCGEncodingMacRoman  );// ok
+    // Uncommented code does not work. Core text uses different coordinates.
+    // CGPoint point = CGPointMake(x, y);
+    // CTFontDrawGlyphs(macFont->fCTFont, &glyph, &point, 1, mContext);
+    ::CGContextSetFont(mContext, macFont->fCGFont);
```

Comment:
this was previously done in Set[Music|Text]Font. Unless the behaviour changed, there is no need to do that each time a character is drawn.

Reply:
I have changed that behaviour. I don't know if this incurs a performance penalty. I think it is cleaner, when using Core Text instead of Core Graphics.